Super-Linter Python Ruff linter errors

```
  2024-11-18 01:21:53 [INFO]   Linting PYTHON_RUFF items...
  Error: -18 01:21:53 [ERROR]   Found errors when linting PYTHON_RUFF. Exit code: 1.
  2024-11-18 01:21:53 [INFO]   Command output for PYTHON_RUFF:
  ------
  config/layout/type.validate.py:22:16: E712 Avoid inequality comparisons to `True`; use `if not item["is_mother"]:` for false checks
     |
  21 |         if "is_mother" in item:
  22 |             if item["is_mother"] != True and item["is_mother"] != False:
     |                ^^^^^^^^^^^^^^^^^^^^^^^^^ E712
  23 |                 print(item)
  24 |                 error("is_mother is invalid in an item")
     |
     = help: Replace with `not item["is_mother"]`
  
  config/layout/type.validate.py:22:46: E712 Avoid inequality comparisons to `False`; use `if item["is_mother"]:` for truth checks
     |
  21 |         if "is_mother" in item:
  22 |             if item["is_mother"] != True and item["is_mother"] != False:
     |                                              ^^^^^^^^^^^^^^^^^^^^^^^^^^ E712
  23 |                 print(item)
  24 |                 error("is_mother is invalid in an item")
     |
     = help: Replace with `item["is_mother"]`
  
  Found 2 errors.
  No fixes available (2 hidden fixes can be enabled with the `--unsafe-fixes` option).
  ------
```

--- a/config/admin/app.py
+++ b/config/admin/app.py
@@ -1,8 +1,7 @@
 import os
-from collections import defaultdict
 
 import yaml
-from flask import Flask, jsonify, redirect, render_template, request, session, url_for
+from flask import Flask, redirect, render_template, request, session, url_for
 
 app = Flask(__name__)
 app.secret_key = "your_secret_key"

--- a/config/module/utils.py
+++ b/config/module/utils.py
@@ -46,7 +46,7 @@ def command_layout_type(item: Dict[str, Any], prefix: str, value: str) -> str:
         + f'"{generate_name}", '
         + f'"{value}", '
         + f"{is_mother}"
-        + f")\n"
+        + ")\n"
     )
 
 

--- a/config/sync.py
+++ b/config/sync.py
@@ -1,7 +1,7 @@
 import json
 import os
 import sys
-from typing import Any, Dict, List
+from typing import Any, Dict
 
 import yaml
 from module.utils import command_layout_style_type, command_layout_type
@@ -48,12 +48,12 @@ def prettify_block_type(item: Dict[str, Any], group: Dict[str, Any]) -> str:
     idtextlower = idtext.lower()
 
     return (
-        f"ADD_BLOCK_TYPE"
-        + f"("
+        "ADD_BLOCK_TYPE"
+        + "("
         + f"{itemid}, "
         + f'"{idtext}", '
         + f'"{idtextlower}"'
-        + f")\n"
+        + ")\n"
     )
 
 
@@ -84,40 +84,40 @@ def prettify_layout_attribute_type(item: Dict[str, Any], group: Dict[str, Any]) 
             if index == 0:
                 result += (
                     key_normal
-                    + f"("
+                    + "("
                     + (
                         f"{itemid}, "
                         + f'"{idtext}", '
                         + f'"{idtextlower}", '
                         + f'"{generate_name}", '
                         + f'"{str(value)}"'
-                        + f")\n"
+                        + ")\n"
                     )
                 )
             else:
                 result += (
                     key_repeat
-                    + f"("
+                    + "("
                     + (
                         f"{itemid}, "
                         + f'"{idtext}", '
                         + f'"{idtextlower}", '
                         + f'"{generate_name}", '
                         + f'"{str(values)}"'
-                        + f")\n"
+                        + ")\n"
                     )
                 )
         return result
     else:
         return (
             key_normal
-            + f"("
+            + "("
             + f"{itemid}, "
             + f'"{idtext}", '
             + f'"{idtextlower}", '
             + f'"{generate_name}", '
             + f'"{str(values)}"'
-            + f")\n"
+            + ")\n"
         )
 
 
@@ -146,11 +146,11 @@ def prettify_layout_attribute_style_global_value(
                 result += key_normal
             else:
                 result += key_repeat
-            result += f"(" + f"{itemid}, " + f'"{str(values)}"' + f")\n"
+            result += "(" + f"{itemid}, " + f'"{str(values)}"' + ")\n"
 
         return result
     else:
-        return key_normal + f"(" + f"{itemid}, " + f'"{str(values)}"' + f")\n"
+        return key_normal + "(" + f"{itemid}, " + f'"{str(values)}"' + ")\n"
 
 
 def prettify_layout_attribute_style_type(
@@ -271,27 +271,27 @@ def prettify_layout_attribute_style_state_type(
                 result += "ADD_LAYOUT_ATTRIBUTE_STYLE_STATE_TYPE_REPEAT"
 
             result += (
-                f"("
+                "("
                 + f"{itemid}, "
                 + f'"{idtext}", '
                 + f'"{idtextlower}", '
                 + f'"{str(value)}", '
                 + f'"{generate_name}"'
-                + f")\n"
+                + ")\n"
             )
             index += 1
 
         return result
     else:
         return (
-            f"ADD_LAYOUT_ATTRIBUTE_STYLE_STATE_TYPE"
-            + f"("
+            "ADD_LAYOUT_ATTRIBUTE_STYLE_STATE_TYPE"
+            + "("
             + f"{itemid}, "
             + f'"{idtext}", '
             + f'"{idtextlower}", '
             + f'"{str(values)}", '
             + f'"{generate_name}"'
-            + f")\n"
+            + ")\n"
         )
 
 
@@ -571,7 +571,6 @@ def generate_document():
             docs += "| نام ویژگی | توضیح | تولید |\n"
             docs += "|-----------|------------|-------|\n"
             for attr in allowed_attributes:
-
                 attr_item = get_a_attr(attributes, attr)
                 # print("attr_item: ", attr_item, attr)
                 # print(attributes)

